// This function is used to provide error message
export const HandleError = () => <div id="errorMessage">Please Try Again</div>;
